gateway-service/src/chef/chef.controller.ts

```
@@ -0,0 +1,136 @@
+  async signUpChef(@Body() body: any): Promise<Observable<any>> {
+    try {
+      const signUp = await this.ChefService.postSignUpChef(body);
+      if (signUp.status === HttpStatus.CONFLICT) {
+        throw new ConflictException(signUp.message);
+      }
+      return new Observable((observer) => {
+        observer.next(signUp);
+        observer.complete();
+      });
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)